_reports/akshatkarani.md

```diff
@@ -0,0 +1,149 @@
+---
+categories:
+  - GSoC
+  - 2019
+  - StatusReport
+date: 2019-08-25
+student: Akshat Karani
+organisation: coala
+organisation_link: https://coala.io
+project: Next Generation Action System
+project_link: https://summerofcode.withgoogle.com/projects/#5450946933424128
+tarball: https://github.com/akshatkarani/GSoC/raw/master/akshatkarani.tar.gz
+sha256sum: 5460a8042082690c2353f487ff9a6fd0a2d4ade1597baaf7e2b74b1633dcd5f6
+mentors: >
+  [Abhinav Kaushlya](https://github.com/abhishalya), [Vamshi Krishna](https://github.com/Vamshi99) & [Kriti Rohilla](https://github.com/kriti21)
+phase:
+  - Bonding : https://gitlab.com/coala/GSoC/gsoc-2019/-/milestones/19
+  - Phase 1 : https://gitlab.com/coala/GSoC/gsoc-2019/-/milestones/20
+  - Phase 2 : https://gitlab.com/coala/GSoC/gsoc-2019/-/milestones/21
+  - Phase 3 : https://gitlab.com/coala/GSoC/gsoc-2019/-/milestones/22
+bio: >
+ I'm a third year student of Computer Science & Engineering at
+ Indian Institute of Technology, Dharwad. I am enthusiatic about
+ contributing to open source and working on my own projects.
+ I participated in GSoC and worked with [coala](https://coala.io) to provide
+ support for bears to define their own custom actions. I also provided support
+ for bears to suggest multiple patches for a problem.
+social:
+  - GitHub:
+      - username: akshatkarani
+      - link: https://github.com/akshatkarani
+  - GitLab:
+      - username: akshatkarani
+      - link: https://gitlab.com/akshatkarani
+  - Gitter:
+      - username: akshatkarani
+      - link: https://gitter.im/akshatkarani
+  - Openhub:
+      - username: akshatkarani
+      - link: https://www.openhub.net/accounts/akshatkarani
+email: akshatkarani@gmail.com
+blog: http://akshatkarani.github.io/blog
+activity:
+ - 1:
+   - repo: cEP
```

Comment:
Follow https://raw.githubusercontent.com/coala/projects/master/_reports/example.md for indentation, etc. You seem to have different indentation. Maybe because of editor?

Reply:
Done